app/services/google_translation_service.rb

```diff
@@ -14,32 +16,43 @@ def initialize
     @cache = Rails.cache
   end
 
-  def translate(text, to)
+  def translate(text:, field_names:, targets:, caller_method:, listing_id: nil)
```

Comment:
Adding a bunch of args here just for better logs isn't ideal, but I think it's the most quickest way to get useful logging information.

Later when we do some more refactoring, we can do a more thorough cleanup of logging.

Reply:
Sure, can you add that as a todo in a comment above the method, that these extra params are only for logging?

Reply:
On second thought - I don't think we should do this. Just log this information before translate is called. So the pattern of the logs could be:
"Calling translate"
"Attempting translation"
"Translation successful"

Reply:
> Just log this information before translate is called

Do you mean moving the logs to the methods that call `translate`, e.g the `translate_event_values` and `process_translations` methods?

Reply:
yes!